projects/core/notifications/send.py

```
@@ -0,0 +1,500 @@
+    if not client:
+        logging.fatal("Couldn't get the slack client ...")
+        return True
+    safe_title = html.escape(title)
+    channel_msg_ts, channel_message = slack_api.search_channel_message(client, safe_title)
+
+    if not channel_msg_ts:
+        channel_message = f"🧵 Thread for `{title}` continuous performance testing"
+        if dry_run:
+            logging.info(f"Posting Slack channel notification ...\n{channel_message}")
+        else:
+            channel_msg_ts, channel_ok = slack_api.send_message(client, message=channel_message)
```

Ruff (0.15.6)

[warning] Unpacked variable `channel_ok` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)